vendor/manifest

```diff
@@ -218,6 +218,14 @@
 			"branch": "master"
 		},
 		{
+			"importpath": "github.com/vmware/photon-controller-go-sdk",
```

Comment:
We should check with photon team if they want us to use master or a different point in time